### DISCUSSION TITLE
Scalene: Program did not run for long enough to profile despite long runtime

### DISCUSSION THREAD

**Discussion by moserb94** [plasma-umass/scalene#615] [Q&A]
Hello,

I am trying to run Scalene for memory profiling a large python code. The program uses ivy-wfengine to execute a series of plugins and runs for roughly 15 minutes. When I try to run` scalene myprogram.py`, the code runs but I end up getting the output `Scalene: Program did not run for long enough to profile.` and an empty json file.

Do you know what could be going wrong? Is this because we call several modules?

Best,
Beatrice

**Comment by drorbar** [comment]
Also, it happens to me when I try to run a function that takes 2 minutes.
I even tried to run the example, but still same error


%%scalene --reduced-profile
# Profile more than one line of code in a cell
x = 0
for i in range(1000):
    for j in range(1000):
        x += 1


Is it possible that this happens because I am running ipynb in vs. code?
